library/spdm_requester_lib/libspdm_req_send_receive.c

```diff
@@ -658,17 +658,8 @@ libspdm_return_t libspdm_receive_spdm_response(libspdm_context_t *spdm_context,
 
         /* These are the expected possible large response types.
          * Anything else is an unexpected error. */
-        if (spdm_response->request_response_code != SPDM_DIGESTS &&
-            spdm_response->request_response_code != SPDM_CERTIFICATE &&
-            spdm_response->request_response_code != SPDM_CHALLENGE_AUTH &&
-            spdm_response->request_response_code != SPDM_MEASUREMENTS &&
-            spdm_response->request_response_code != SPDM_VENDOR_DEFINED_RESPONSE &&
-            spdm_response->request_response_code != SPDM_ALGORITHMS &&
-            spdm_response->request_response_code != SPDM_KEY_EXCHANGE_RSP &&
-            spdm_response->request_response_code != SPDM_FINISH_RSP &&
-            spdm_response->request_response_code != SPDM_PSK_EXCHANGE_RSP &&
-            spdm_response->request_response_code != SPDM_ENCAPSULATED_REQUEST &&
-            spdm_response->request_response_code != SPDM_ENCAPSULATED_RESPONSE_ACK
+        if (spdm_response->request_response_code == SPDM_VERSION ||
```

Comment:
There are other responses that should not be chunked, like `HEARTBEAT_ACK`.

Reply:
I don't think we need deal with HEATBEAT_ACK.

The check is based upon SPDM spec, which mentioned that VERSION and CAPABILITIES shall not be chunked. HEATBEAT_ACK is not listed there.